include/gsl/multi_span

```diff
@@ -79,7 +79,7 @@ namespace details
     template <typename SizeType>
     struct SizeTypeTraits
     {
-        static const SizeType max_value = std::numeric_limits<SizeType>::max();
+        static const SizeType max_value = (std::numeric_limits<SizeType>::max)();
```

Comment:
I think the problem should be fixed outside the source file -- not necessarily by code removal.